lucetc/tests/wasm.rs

```diff
@@ -508,14 +528,76 @@ mod module_data {
         );
         */
     }
+
+    #[test]
+    fn names_local() {
```

Comment:
I always like to point people towards the tests first, so _here_ there are! :microscope: :woman_scientist: :point_down: